Tests/IntegrationTest.php

```diff
@@ -313,7 +478,7 @@ class IntegrationTestKernel extends Kernel
 
     public function __construct(string $environment, bool $debug)
     {
-        $this->randomKey = rand(100, 999);
+        $this->randomKey = hash('sha3-384', spl_object_hash($this) . self::$invocations++);
```

Comment:
What is this about?

Reply:
When running the suite before, 1 out of 1000 times I got weird errors, because it reused a previous container. This makes sure that this never happens

Reply:
Please move this to a separate commit that will contain this explanation, and take it as an occasion to squash everything else into a single commit.